sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -1349,7 +1349,13 @@ public List<PersonIndexDto> getIndexList(PersonCriteria criteria, Integer first,
 			person.get(Person.CHANGE_DATE),
 			JurisdictionHelper.booleanSelector(cb, service.inJurisdictionOrOwned(personQueryContext)));
 
-		cq.where(person.get(Person.ID).in(indexListIds));
+		Predicate filter = person.get(Person.ID).in(indexListIds);
```

Comment:
Why is this needed? This indexListFilter is already part of the getIndexListIds logic

Reply:
`indexListFilter` has been reintroduced in commit 421860c after failure of test `PersonFacadeEjbUserFilterTest.testGetPersonIndexListWhenSeveralAssociations`, where a person entry is duplicated with different values for `PersonIndexDto.isInJurisdiction` (see https://github.com/hzi-braunschweig/SORMAS-Project/actions/runs/3429250031/jobs/5714616024). In this case, one of these two distinct entries is filtered out by the `indexListFilter`.